unity-client/Assets/Scripts/MainScripts/DCL/Configuration/Configuration.cs

```diff
@@ -22,6 +24,24 @@ public static class BuilderInWorldSettings
         public const string STATE_EVENT_NAME = "stateEvent";
         public const string SCENE_EVENT_NAME = "SceneEvent";
         public static float ENTITY_POSITION_REPORTING_DELAY = 0.1f; // In seconds
+
+
+        //Floor Scene Object
+        public const string FLOOR_ID = "da1fed3c954172146414a66adfa134f7a5e1cb49c902713481bf2fe94180c2cf";
+        public const string FLOOR_MODEL = "FloorBaseGrass_01/FloorBaseGrass_01.glb";
+        public const string FLOOR_NAME = "Floor";
+
+        public const string FLOOR_GLTF_KEY = "FloorBaseGrass_01/FloorBaseGrass_01.glb";
+        public const string FLOOR_GLTF_VALUE = "QmSyvWnb5nKCaGHw9oHLSkwywvS5NYpj6vgb8L121kWveS";
+
+        public const string FLOOR_TEXTURE_KEY = "FloorBaseGrass_01/Floor_Grass01.png.png";
+        public const string FLOOR_TEXTURE_VALUE = "QmT1WfQPMBVhgwyxV5SfcfWivZ6hqMCT74nxdKXwyZBiXb";
+
+
+        //Collectables
+        public const string ASSETS_COLLECTIBLES = "Collectibles";
+        public const string COLLECTIBE_MODEL_PROTOCOL = "ethereum://";
+    
```

Comment:
I cannot find references for most of this consts. Remove them if they are not needed.

Reply:
Every const in this file has a reference.